data/remnant/remnant ships.txt

```diff
@@ -892,37 +892,43 @@ ship "Merganser"
 		category "Interceptor"
 		licenses
 			Remnant
-		"cost" 6830000
-		"shields" 3000
-		"hull" 5000
+		"cost" 4830000
+		"shields" 3500
+		"hull" 3000
 		"required crew" 1
 		"bunks" 3
 		"cargo space" 11
 		"fuel capacity" 600
 		"ramscoop" 1.75
-		"mass" 30
-		"drag" 1.5
+		"mass" 90
+		"drag" 1.3
 		"heat dissipation" .9
 		"gaslining" 1
-		"outfit space" 219
-		"shield generation" 2.8
+		"outfit space" 181
+		"weapon capacity" 60
+		"engine capacity" 60
+		"shield generation" 1.4
 		"shield energy" 1.2
-		"hull repair rate" 1.8
+		"hull repair rate" 0.9
 		"hull energy" 0.6
-		"cloak" .04
-		"cloaking energy" 2
-		"cloaking fuel" .05
+		"cloak" .03
+		"cloaking energy" 4
+		"cloaking fuel" .075
 		"atmosphere scan" 100
 		"outfit scan power" 12
 		"outfit scan efficiency" 12
 		"tactical scan power" 26
-		"ion resistance" 0.1
-		"ion resistance energy" -0.05
-		"scramble resistance" 0.1
-		"scramble resistance energy" -0.05
+		"ion resistance" 0.025
+		"scramble resistance" 0.025
```

Comment:
I tested the Merganser’s ability to recover from ember waste storms.

Generally, the storms happen on average every 2.5 minutes or so, so the 45 second recovery time is not an issue, especially as the recovery time is the same for power as well, and power recovery is faster than on master. 

Out of ten trials, the merganser was able to fully recover 70% of the time, which matches the theoretical estimate of 45s/150s=30% of the time having a storm happen again before the ship can recover. However, in those trials, it depleted to about 85% health on average, so the theoretical estimate is probably more like 75%.

Increasing the ion resistance to 0.1 results in around an 85% full recovery, with it being depleted to around 50% health on average, but I don’t think that matters too much. Generally, it took me a while of waiting on 3x speed to even get an ion storm at all, so most casual players probably wouldn’t notice the issue other than one or two times. It shouldn’t become an annoyance either way, so I think the ion resistance can be nerfed.

The main justification for the nerf seems to be that the peregrine has around 4x the mass, so if ion resistance scales with mass, as it does for the outfits, the peregrine would have 0.4 ion resistance, but it has 0.15 ion resistance instead, which appears to not be considered an issue. 0.15/4 is 0.0375, though, so the ion resistance could maybe be raised a little bit:

```suggestion
		"ion resistance" 0.035
		"scramble resistance" 0.035
```

This is a conservative value as that is how the outfits appeared to round, and because the Peregrine is an improvement over the Merganser, though I’m really fine with any value from 0.025 to 0.1. This is mostly a consistency thing, not something that would really be noticeable in game. 

This is the last of the concerns that I saw raised in this PR that I hadn’t addressed, so my approval stands.